spec/models/user_spec.rb

```diff
@@ -14,4 +14,38 @@
     expect(user).to_not be_valid
     expect(user.errors[:password_confirmation]).to_not be_blank
   end
+
+  it 'should calculate the appropriate numbers' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.